test/blackbox/run_blackbox_tests/test_optimal_pmtu.h

```diff
@@ -0,0 +1,62 @@
+#ifndef TEST_CASES_OPTIMAL_PMTU_H
+#define TEST_CASES_OPTIMAL_PMTU_H
+
+/*
+    test_optimal_pmtu.h -- Declarations for Individual Test Case implementation functions
+    Copyright (C) 2019  Guus Sliepen <guus@meshlink.io>
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+#include <stdbool.h>
+
+extern int test_optimal_pmtu(void);
+extern int total_tests;
+extern char *lxc_path;
```

Comment:
We shouldn't need a forward declaration for `lxc_path` here. Instead, any `.c` file that uses `lxc_path` should instead have a `#include "../common/containers.h"`.